src/ASM/ASMbase.h

```diff
@@ -599,6 +599,17 @@ class ASMbase
   //! \param[in] silence If \e true, suppress debug print
   bool addMPC(MPC*& mpc, int code = 0, bool silence = false);
 
+  //! \brief Element error and associated index.
+  //! \note The error value must be first and the index second, such that the
+  //! internally defined greater-than operator can be used when sorting the
+  //! error+index pairs in decreasing error order.
+  typedef std::pair<double,int> DblIdx;
+
+  //! \brief Remap element wise errors from geometry mesh to refinement mesh.
+  //! \param     errors The remapped errors
+  //! \param[in] patch The patch which holds the meshes
+  //! \param[in] origErr The element wise errors on the geometry mesh
+  virtual void remapErrors(std::vector<DblIdx>& errors, const Vector& origErr);
```

Comment:
Does the default version need to do anything at all, i.e., will it ever be invoked for the non-LR sub-classes?
If the answer is "no", the empty method can be inlined with only anonymous arguments.
If yes, only remove the non-existing `patch` argument from doxy.

Reply:
This is changed in the hybrid between this and multipatch refinements so i will ignore this one. The answer is that yes, we transfer between data structures so it has to do things. Doxy is fixed in hybrid.